deployment/README.md

```diff
@@ -1,60 +1,133 @@
+
 ### Build
-1. Download the [Dockerfile](./sample-workload/Dockerfile) for building TDX demo workload
+1. Download the [Dockerfile](./sample-workload/Dockerfile) for building the Intel TDX demo workload.
    ```bash
    wget https://raw.githubusercontent.com/arvind5/trustauthority-samples/task/dockerfile-deployment/deployment/sample-workload/Dockerfile
    ```
-2. Run the following command to build the Docker image for the TDX demo workload
+2. Run the following command to build the Docker image for the TDX demo workload.
    ```bash
    sudo docker build --build-arg PLATFORM=dcap -t trustauthority-demo .
    ```
-   Where `PLATFORM` is the underlying TDVM where you want to run the workload, e.g., azure, gcp, dcap (Intel Dev Cloud)
+   The demo workload relies on the Intel Trust Authority CLI for Intel TDX to obtain a quote and request an attestation token. The CLI requires an adapter that is specific to the platform on which the demo app is run. The 'PLATFORM' argument specifies which adapter to use:
+
+   | PLATFORM | Description |
+   |---|---|
+   | `dcap` | Installs the adapter for native Intel TDX TDs that use Intel SGX DCAP to collect a quote for attestation.|
+   | `azure` | Installs the adapter for Microsoft Azure confidential VMs with Intel TDX. |
+   | `gcp` | Installs the adapter for Google Cloud Platform confidential VMs. |
 
 ### Deploy
-1. Download the [workload.env](./sample-workload/workload.env) file and update the missing configurations
+1. Download the [workload.env](./sample-workload/workload.env) file and update the missing configurations.
    ```bash
    wget https://raw.githubusercontent.com/arvind5/trustauthority-samples/task/dockerfile-deployment/deployment/sample-workload/workload.env
    ```
-2. Run the following command to start the TDX demo workload as a Docker container
+   The **workload.env** file contains the settings needed for the sample workload to communicate with Intel KBS and Intel Trust Authority. This file contains secrets. Embedding secrets in an unencrypted file is _not_ a recommended best practice for secure systems. It's done here to make the demo easy to configure.
+
+   The `KBS_ADMIN` and `KBS_PASSWORD` must match the `ADMIN_USERNAME` and `ADMIN_PASSWORD` used in KBS.env.
+
+   The `KBS_URL` is of the form `https://<IP_address>:9443/kbs/v1`, where IP_address is the IP address of the host machine where the relying-party (KBS) container is running. If the KBS is running on same TDVM as the workload, then provide `172.17.0.1` as IP_address. Port 9443 is the default listening port for Intel KBS. If you change the port number for the Intel KBS container, you'll need to make a corresponding change to `KBS_URL`.
+
```

Comment:
```suggestion
SKIP_TLS_VERIFICATION if set to **true** will skip the TLS server certificate verification. For this demo, set it to **true** since we are using self-signed certificates for TLS.
```